README.md

```diff
@@ -54,11 +54,13 @@ Please install these tools using pip:
 
     pip install gftools
 
+(Mac OS users on M1 chips will need to upgrade pip first to support universal wheels: `pip install -U pip`.)
```

Comment:
doesn't it depend on where they get their python from, though? If one installs, say, 3.10 from Homebrew or Python.org, I bet they do have pip up to date. Perhaps it's just the python that 's available from Apple that is out of date? Just speculating as I am not on M1 yet.

Reply:
You're probably technically right, but the point is to give people a nice clear instruction to head off problems later. There's no real harm in people upgrading pip if they're using a non-Apple Python, and it makes the instructions easier to follow if we say Just Do It.

Reply:
yeah, upgrading pip is the first thing I always do myself (along with setuptools and wheel) whenever I create a new venv